src/v2/Apps/Feature/Components/FeatureHeader/FeatureHeaderDefault.tsx

```diff
@@ -45,46 +42,51 @@ export interface FeatureHeaderDefaultProps {
 export const FeatureHeaderDefault: React.FC<FeatureHeaderDefaultProps> = ({
   feature: { name, subheadline, defaultImage: image },
 }) => {
-  return (
-    <Container
-      display={["block", "flex"]}
-      height={[
-        "auto",
-        !!image ? `calc(95vh - ${DESKTOP_NAV_BAR_HEIGHT}px)` : "50vh",
-      ]}
-    >
-      {image && (
-        <Figure height={["50vh", "auto"]}>
-          <picture>
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <source srcSet={image.sm.srcSet} media="(max-width: 400px)" />
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <source srcSet={image.md.srcSet} media="(max-width: 1200px)" />
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <source srcSet={image.lg.srcSet} media="(min-width: 1200px)" />
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <Image src={image.sm.src} alt={name} loading="lazy" />
-          </picture>
-        </Figure>
-      )}
+  if (image) {
+    return (
+      <FullBleed>
+        <Container
+          display={["block", "flex"]}
+          height={[
+            "auto",
+            !!image ? `calc(95vh - ${DESKTOP_NAV_BAR_HEIGHT}px)` : "50vh",
+          ]}
+        >
+          <Figure height={["50vh", "auto"]} backgroundColor="black10">
+            <picture>
+              {/* @ts-expect-error STRICT_NULL_CHECK */}
```

Comment:
👋 not sure if you've come across these `ts-expect-error`s before, but here's a [little history](https://github.com/artsy/force/blob/master/docs/best_practices.md#avoid-copying-and-try-to-fix--ts-expect-error-strict_null_check-flags). 

If you ever see them when working on something its super helpful to try to cleanup -- usually its as easy as adding a guard or a `?` between props chaining 🙏

Reply:
thanks for the heads up, I was wondering about them and had no clue about the history.
Gonna fix clean them 🙏

Reply:
removed a few ones in this [commit](https://github.com/artsy/force/pull/8501/commits/ab72d5a946ce1d92266a284c76ef4aa9fcbf610d), the other that I didn't touch looked a bit more tricky, probably with some nested NonNullable types, please let me know if I should fix them here in this PR 😄

Reply:
Thanks for taking care of those 👍 

As a general rule, if they're easy to fix, then we should fix (think, adding a `?` to make object access safe) but if the error is not clear and you end up wading around in the weeds then def don't worry about it.